tests/lib/collect-features.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+after_non_nested_task() {
+    # Write to the directory specified in the spread.yaml file for artifacts
+    local write_dir="${SPREAD_PATH}/feature-tags"
+    local task_dir="${write_dir}/${SPREAD_JOB//\//--}"
+    mkdir -p "$write_dir"
+    mkdir -p "$task_dir"
+    "$TESTSTOOLS"/journal-state get-log -u snapd --no-pager --output cat > "$task_dir"/journal.txt
```

Comment:
we should filter features lines here, so just save the lines we need to post process.